Source/VungleAdapter.swift

```diff
@@ -161,20 +143,44 @@ final class VungleAdapter: PartnerAdapter {
             return nil
         }
 
-        let vungleErrorCode = VungleSDKErrorCode(rawValue: errorCode)
-
-        switch vungleErrorCode {
-        case VungleSDKErrorNoAppID:
+        switch errorCode {
+        case 2: // The app ID fails SDK validation; for example, an empty string.
             return .initializationFailureInvalidCredentials
-        case VungleSDKErrorInvalidiOSVersion:
-            return .initializationFailureOSVersionNotSupported
-        case VungleSDKErrorSDKAlreadyInitializing:
-            return .initializationFailureAborted
+        case 3: // The SDK was already initializing when another call is made.
+            return .initializationSkipped
+        case 4: // The SDK was already successfully initialized when another call is made.
+            return .initializationSkipped
+        case 6: // This error is returned if any public API is called before initialization when initialization is required.
```

Comment:
Vungle does not provide constants for these code? Ughh

Reply:
The [documentation](https://support.vungle.com/hc/en-us/articles/15661036969627-Error-Codes-Vungle-SDK-for-iOS#error-case-currentlyinitializing-3-0-1) makes it look like these are an enum internally, but when I search through the public interface for VungleAdsSDK they're not in there.